i18n/create_messages.py

```diff
@@ -30,8 +30,8 @@
 
 def string_is_ascii(s):
   try:
-    s.decode('ascii')
-    return True
+    # This approach is better for compatibility 
+    return all(ord(c) < 128 for c in s)
   except UnicodeEncodeError:
     return False
```

Comment:
Will `all(ord(c) < 128 for c in s)` raise a `UnicodeEncodeError` or can this try block be removed?

Reply:
I think this could be removed.

Reply:
I think you're right, I've removed it.